app/src/main/res/drawable/continent_central_america.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="136.3652"
+    android:viewportHeight="125.4169"
+    android:width="136.36523dp"
+    android:height="125.41689dp">

```

Comment:
These should be `64dp` if I remember correctly. Bonus points for using `64px` in inkscape already and centering the drawing in it.